pages.ne/android/bugreportz.md

```diff
@@ -0,0 +1,21 @@
+# Bugreport
+
+> जिप गरिएको एन्ड्रोइड बग रिपोर्ट उत्पन्न गर्नुहोस्।
+> यो आदेश `adb shell` मार्फत मात्र प्रयोग गर्न सकिन्छ।
+> थप जानकारी: <https://cs.android.com/android/platform/superproject/+/main:frameworks/native/cmds/bugreportz>।
```

Comment:
```
> थप जानकारी: <https://cs.android.com/android/platform/superproject/+/main:frameworks/native/cmds/bugreportz>.
```

Shouldn't it be like this?

Reply:
Nice catch, the page descriptions use the native period whereas more info link one used the conventional period.

Reply:
```suggestion
> थप जानकारी: <https://cs.android.com/android/platform/superproject/+/main:frameworks/native/cmds/bugreportz>.
```